api/models/User.js

```diff
@@ -31,4 +40,100 @@ userSchema.statics.create = function(name) {
 	});
 };
 
+userSchema.statics.read = function(GoogleID)
+{
+	return new Promise((resolve , reject) => {
+		mongoose.model('User', userSchema).findOne({googleID: GoogleID}, (err,user)=>{
+			if(err) reject(err);
+			else resolve(user);
+		});
+	});
+}
+
+userSchema.statics.delete = function(GoogleID)
+{
+	return new Promise((resolve, reject)=>{
+		mongoose.model('User', userSchema).findOne({googleID: GoogleID}).remove((err,offer)=>{
+			if(err) reject(err);
+		});
+	});
+}
+
+
+// Static: User.doSomething
+// Instance method: 
+// var user;
+// user.doSomething()
+
+
+userSchema.methods.setisAdminTo = function(becomeAdmin)
```

Comment:
The function could be named better to maybe `setAdminStatus`. The parameter name could be named better. Maybe `adminStatus`?